samples/data-grid/basic/simple/demo.css

```diff
@@ -1,12 +1,12 @@
-@import 'https://code.highcharts.com/css/highcharts.css';
-@import 'https://code.highcharts.com/css/datagrid.css';
+@import "https://code.highcharts.com/css/highcharts.css";
+@import "https://code.highcharts.com/css/datagrid.css";
 
-body {
-	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+#container {
+    height: 402px;
```

Comment:
402px ? Interesing, what is the reason of the extra 2px ;D

Reply:
We have 16 rows, each 25px which gives 400px.
I presume that that extra 2px ale for borders at the top and bottom.